assetfs.go

```diff
@@ -137,7 +144,11 @@ func (fs *AssetFS) Open(name string) (http.File, error) {
 		name = name[1:]
 	}
 	if b, err := fs.Asset(name); err == nil {
-		return NewAssetFile(name, b), nil
+		timestamp := defaultFileTimestamp
```

Comment:
You could add something like this here, to prompt people to re-generate:

```Go
if fs.AssetInfo == nil {
    log.Fatalln("go-bindata-assetfs: you're using a new version of the library but haven't re-generated the file, so it's not compatible; regenerate to avoid this problem")
}
```

Or something like that.